### ISSUE TITLE
pkg_resources is deprecated

### ISSUE THREAD

**Issue by bdauvergne** [akretion/factur-x#38]
```
lib/python3.11/site-packages/facturx/facturx.py:40: DeprecationWarning: pkg_resources is deprecated as an API. See
https://setuptools.pypa.io/en/latest/pkg_resources.html
    from pkg_resources import resource_filename
```

https://setuptools.pypa.io/en/latest/pkg_resources.html

**Comment by magicmatt007** [comment]
When trying to run the command line tools on my Ubuntu LTS 24 I get the message 

`"ModuleNotFoundError: No module named 'pkg_resources'"`

I guess it is related to this issue above.

How to fix it?

**Comment by bdauvergne** [comment]
It's fixed with #39.
